docs/_global_substitutions.py

```diff
@@ -122,6 +122,7 @@
 
 links_to_become_subs: dict[str, str] = {
     "Astropy": "https://docs.astropy.org",
+    "Astropy Equivalencies": "https://docs.astropy.org/en/stable/units/equivalencies.html",
```

Comment:
👍🏻 Agreed that this should be a global sub. (And turns out there isn't a submarine emoji.)